deploy/installer/gui/installscript.qs

```diff
@@ -23,6 +23,18 @@ Component.prototype.createOperations = function()
             component.addOperation( "CreateShortcut", "@TargetDir@\\nheko.exe", "@DesktopDir@\\nheko.lnk",
                                     "workingDirectory=@TargetDir@", "iconPath=@TargetDir@\\nheko.exe",
                                     "iconId=0", "description=Desktop client for the Matrix protocol");
+
+            var reg = installer.environmentVariable("SystemRoot") + "\\System32\\reg.exe";
```

Comment:
I guess it adds it to the 32bit registery? Does that matter though?
And I think the installer is 32bit, not sure.

Reply:
If it is in the 32bit registry, it may not find nheko. Someone reported issues regarding that. But if you say it works fine, then this is okay :3

Reply:
Yeah it adds it properly for me (I opened the normal registry editor and could see it there), and this is added to hkey_classes_root which I think is common for both?